firstUniqueElement

```diff
@@ -0,0 +1,9 @@
+const firstUniqueElement = text => {
+  const arrayOfText = text.split('');
+
+  for (let i = 0; i < text.length; i++) {
```

Comment:
This is correct, but you need to figure out that this works many times, so for any array item you loop it 3 times with indexOf. Find a solution with one iteration